lib/csv.rb

```diff
@@ -1352,36 +1352,51 @@ def parse_line(line, **options)
     end
 
     #
-    # Use to slurp a CSV file into an Array of Arrays. Pass the +path+ to the
-    # file and +options+.
-    # See {Options for Parsing}[#class-CSV-label-Options+for+Parsing].
-    #
-    # This method also understands
-    # an additional <tt>:encoding</tt> parameter that you can use to specify the
-    # Encoding of the data in the file to be read. You must provide this unless
-    # your data is in Encoding::default_external(). CSV will use this to determine
-    # how to parse the data. You may provide a second Encoding to have the data
-    # transcoded as it is read. For example,
-    # <tt>encoding: "UTF-32BE:UTF-8"</tt> would read UTF-32BE data from the file
-    # but transcode it to UTF-8 before CSV parses it.
+    # :call-seq:
+    #   read(source, **options ) -> new_csv
```

Comment:
```suggestion
    #   read(source, **options) -> new_csv
```

Reply:
It doesn't return `new_csv`. 
It returns an array of rows or `CSV::Table`.

Reply:
Ook!  When I typed the call-seq, I didn't know what would be returned.  Then forgot to go back.
Both fixed in next  push.